src/components/cards/GenericCard.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, AccentColorProvider } from '@/design-system';
+import React from 'react';
+import { ButtonPressAnimation } from '../animations';
+import LinearGradient from 'react-native-linear-gradient';
+import { globalColors } from '@/design-system/color/palettes';
+import { deviceUtils } from '@/utils';
+import ConditionalWrap from 'conditional-wrap';
+
+// (device width - (horizontal inset * 2 + padding between cards)) / # of cards in row
+export const SQUARE_CARD_SIZE = (deviceUtils.dimensions.width - 60) / 2;
```

Comment:
Might be a dumb question, sorry: why does this need to exist? We can't just do like 50%?

Reply:
if the card sizes are 50% of the container width, they will bump against each other with no padding between. i need to account for the padding between them too, which is 20 in this case.